lib/DueDatePicker/styles.scss

```diff
@@ -0,0 +1,349 @@
+/* ==========================================================================
+   Config
+   ========================================================================== */
+$duedate-popover-width: 304px !default;
+$duedate-timepicker-width: 80px !default;
+$duedate-timepicker-height: 220px !default;
+/* ==========================================================================
+   Styles
+   ========================================================================== */
+.duedate__popover.duedate__popover {
+  max-width: $duedate-popover-width;
+  width: $duedate-popover-width;
+  margin-top: $layout-spacing-base / -2;
+
+  .arrow {
+    display: none;
+  }
+
+  &.duedate__popover .popover-content {
+    padding: 0;
+  }
+}
+
+.duedate__container {
+  font-size: $typo-size-slight;
+  position: relative;
+}
+
+.duedate__button {
+  color: $color-text-light;
+
+  .button {
+    text-decoration: underline;
+    color: $color-text-light;
+  }
+
+  .icon {
+    margin-right: $layout-spacing-base / 4;
+    top: -2px;
+    position: relative;
+    vertical-align: middle;
+  }
+
+  .icon path {
+    fill: $color-link-base;
+  }
+
+  &:hover, &:active {
+    text-decoration: none;
+
+    .icon path {
+      fill: $color-link-hover;
+    }
+  }
+}
+
+.duedate__button--set-duedate {
+  color: $color-link-base;
+}
+
+.duedate__button--overdue {
+  color: $primary-red;
+
+  .button {
+    color: $primary-red;
+  }
+
+  &:hover {
+    color: $primary-red;
+
+    .icon path {
+      fill: $primary-red;
+    }
+  }
+
+  .icon path {
+    fill: $primary-red;
+  }
+
+  .icon {
+    top: -1px;
+  }
+}
+
+.duedate__container--completed {
+  .duedate__button, .duedate__toggle {
+    text-decoration: line-through;
+    font-style: italic;
+    color: $color-text-light;
+  }
+
+  .icon path, &:hover .icon path {
+    fill: $color-text-light;
+  }
+}
+
+.duedate__header {
+  padding: $layout-spacing-base / 2;
+  border-bottom: 1px solid $neutral-light;
+  font-size: $typo-size-slight;
+}
+
+.duedate__header--date {
+  font-size: $typo-size-base;
+}
+
+.duedate__header--time {
+  color: $color-text-base;
+  text-decoration: underline;
+
+  &:hover {
+    color: $color-text-base;
+  }
+}
+
+.duedate__header--overdue .duedate__header--date {
+  color: $primary-red;
+  .duedate__header--time {
+    color: $primary-red;
+  }
+}
+
+.duedate__submit {
+  text-align: center;
+  padding: 0 0 $layout-spacing-base / 2;
+}
+
+.duedate__submit--cancel {
+  font-size: $typo-size-slight;
+  margin-right: $layout-spacing-base / 2;
+  color: $neutral-dark;
+
+  &:hover {
+    text-decoration: none;
+    color: $neutral-dark;
+  }
+}
+
+.duedate__submit--save {
+  font-size: $typo-size-slight;
+  padding: $layout-spacing-base / 4 $layout-spacing-base / 2.5;
+}
+
+/* DueDate Calendar */
+
+.duedate__popover .DayPicker {
+  padding: $layout-spacing-base / 2;
+
+  &:after {
+    content: "";
+    display: table;
+    clear: both;
+  }
+
+  .DayPicker-wrapper {
```

Comment:
Is `DayPicker` is provided by the 3rd party? If so I wonder if this should be handled in the overrides styling area? What do you think?